src/test/java/com/devonfw/sample/archunit/LayerDependencyRulesTest.java

```diff
@@ -0,0 +1,97 @@
+package com.devonfw.sample.archunit;
+
+import com.tngtech.archunit.core.importer.ImportOption;
+import com.tngtech.archunit.junit.AnalyzeClasses;
+import com.tngtech.archunit.junit.ArchTest;
+import com.tngtech.archunit.lang.ArchRule;
+
+import static com.tngtech.archunit.lang.syntax.ArchRuleDefinition.classes;
+import static com.tngtech.archunit.lang.syntax.ArchRuleDefinition.noClasses;
+
+@AnalyzeClasses(packages = "com.devonfw.sample.archunit", importOptions = ImportOption.DoNotIncludeTests.class)
+public class LayerDependencyRulesTest {
+    
+    // 'access' catches only violations by real accesses, i.e. accessing a field, calling a method; compare 'dependOn' further down
+
+    // L09: verifying that code from logic layer does not access service layer (of same app).
+    @ArchTest
+    static final ArchRule logic_should_not_access_services =
+            noClasses().that().resideInAPackage("..logic..")
+                    .should().accessClassesThat().resideInAPackage("..service..");
```

Comment:
you should follow the code sample I provided in the story with `mayOnlyBeAccessedByLayers` to utilize the concept of layer-checks in arch-unit.

Reply:
Thank you for the hint!

Reply:
Thanks, done and resolved.